openedx/adg/lms/webinars/helpers.py

```diff
@@ -30,9 +33,15 @@ def send_webinar_emails(template_slug, webinar, recipient_emails, send_at=None):
         'webinar_id': webinar.id,
         'webinar_title': webinar.title,
         'webinar_description': webinar.description,
-        'webinar_start_time': webinar.start_time.strftime("%B %d, %Y %I:%M %p %Z")
+        'webinar_start_time': webinar.start_time.strftime("%B %d, %Y %I:%M %p %Z"),
+        'webinar_link': webinar.meeting_link
     }
 
+    if template_slug == MandrillClient.WEBINAR_CREATED:
+        root_url = configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
+        webinar_description_url = reverse('webinar_event', kwargs={'pk': webinar.id})
+        context['register_link'] = f'{root_url}/login?next={webinar_description_url}?status=register'
```

Comment:
Can we add a helper, Maybe named as `get_webinar_description_link` and use that in tests as well. This is duplicated in tests and we will have to manage both.